### ISSUE TITLE
Ruby 2.2 support

### ISSUE THREAD

**Issue by macler** [larsch/ocra#93]
Ocra doesn't support ruby 2.2.
Any plans?

**Comment by lmayorga1980** [comment]
:+1: 

Having the following error after upgrading from **2.1.5**

```
C:/Ruby22/lib/ruby/2.2.0/Win32API.rb:16:in `initialize': no implicit conversion from nil to integer (TypeError)
        from C:/Ruby22/lib/ruby/2.2.0/Win32API.rb:16:in `new'
        from C:/Ruby22/lib/ruby/2.2.0/Win32API.rb:16:in `initialize'
        from C:/Ruby22/lib/ruby/gems/2.2.0/gems/ocra-1.3.4/bin/ocra:951:in `new'
        from C:/Ruby22/lib/ruby/gems/2.2.0/gems/ocra-1.3.4/bin/ocra:951:in `loaded_dlls'
        from C:/Ruby22/lib/ruby/gems/2.2.0/gems/ocra-1.3.4/bin/ocra:975:in `detect_dlls'
        from C:/Ruby22/lib/ruby/gems/2.2.0/gems/ocra-1.3.4/bin/ocra:840:in `build_exe'
        from C:/Ruby22/lib/ruby/gems/2.2.0/gems/ocra-1.3.4/bin/ocra:1174:in `block in <top (required)>'
```

**Comment by InterpolKun** [comment]
Hey, bro, ruby 2.2 support is very necessary!

**Comment by larsch** [comment]
Fixed in 1.3.5
